src/Tronkell/Server/Server.hs

```diff
@@ -42,8 +42,13 @@ startServer gConfig = do
   -- start network
   wsThread  <- forkIO $ W.start firstUId networkChans clientsChan
   tcpThread <- forkIO $ Tcp.start firstUId networkChans clientsChan
+  -- to avoid space msg leak..
+  forkIO . forever . readChan $ clientSpecificOutChan
+  forkIO . forever . readChan $ clientsChan
+  forkIO . forever . readChan $ internalChan
```

Comment:
there is something like : https://hackage.haskell.org/package/stm-2.4.4.1/docs/Control-Concurrent-STM-TChan.html#v:newBroadcastTChan
and 
there is something like https://hackage.haskell.org/package/broadcast-chan
quoting the library :
```
A variation of Control.Concurrent.Chan from base, which allows to the easy creation of broadcast channels without the space-leaks that may arise from using Control.Concurrent.Chan.dupChan. The Control.Concurrent.Chan.Chan type from Control.Concurrent.Chan consists of both a read and write end. This presents a problem when one wants to have a broadcast channel that, at times, has zero listeners. To write to a Control.Concurrent.Chan.Chan there must always be a read end and this read end will hold ALL messages alive until read. The simple solution applied in this module is to separate read and write ends. As a result, any messages written to the write end can be immediately garbage collected if there are no active read ends, avoding space leaks.
```

but they both required some more work then these 3 lines.